packages/flutter/lib/src/material/slider.dart

```diff
@@ -73,11 +73,15 @@ class _SliderRenderObjectWidget extends LeafRenderObjectWidget {
 }
 
 const double _kThumbRadius = 6.0;
-const double _kThumbRadiusDisabled = 3.0;
+const double _kActiveThumbRadius = 9.0;
+const double _kThumbRadiusDisabled = 4.0;
```

Comment:
Sorry to nit pick but it seems like this constant should be _kDisabledThumbRadius since the other one is _kActive...